package.json

```diff
@@ -17,7 +17,7 @@
     "start-dev": "npm-run-all --parallel watch start",
     "build": "npm-run-all build:*",
     "build:font-awesome": "node scripts/build-fontawesome.js",
-    "build:webpack": "webpack",
+    "build:webpack": "webpack --progress",
```

Comment:
Is this a side-effect of the rest of this PR, or just something you've added along the way?